pkg/fleet-manager/fleet_plugin_backup.go

```diff
@@ -0,0 +1,273 @@
+/*
+Copyright Kurator Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fleet
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
+	"helm.sh/helm/v3/pkg/kube"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"kurator.dev/kurator/pkg/apis/fleet/v1alpha1"
+	"kurator.dev/kurator/pkg/fleet-manager/plugin"
+	"kurator.dev/kurator/pkg/infra/util"
+)
+
+const (
+	AccessKey = "access-key"
+	SecretKey = "secret-key"
+
+	AWS         = "aws"
+	HuaWeiCloud = "huaweicloud"
+	GCP         = "gcp"
+	Azure       = "azure"
+
+	AWSObjStoreSecretName         = "kurator-velero-s3"
+	HuaWeiCloudObjStoreSecretName = "kurator-velero-obs"
```

Comment:
do we need fixed name? Not quite sure we used fleet name + plugin name before

Reply:
it seems that we never create a secret in fleet cluster, while we can add  fleet name + plugin name infomation.